queries.sql

```diff
@@ -7,4 +7,34 @@ select date_of_birth from animals where  name = 'Agumon' or name = 'Pikachu';
 select name, escape_attemps from animals where weight_kg > 10.5;
 select * from animals where neutered = true;
 select * from animals where name not like 'Gabumon';
-select * from animals where weight_kg between 10.4 and 17.3;
\ No newline at end of file
+select * from animals where weight_kg between 10.4 and 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '20220101';
+savepoint sp1;
```

Comment:
- [ ] Change this query from date_of_birth > '20220101' to `date_of_birth >= '2022/01/01`